xen/arch/x86/Kconfig

```diff
@@ -23,6 +23,8 @@ config X86
 	select HAS_PDX
 	select NUMA
 	select VGA
+	select HAS_PM
+	select HAS_CPU_TURBO
```

Comment:
why HAS_CPU_TURBO is turned on here? is there any condition we can check if turbo is really supported by the platform?

Reply:
I just retain original commit https://lists.xen.org/archives/html/xen-devel/2014-11/msg00942.html
Xen community seems to OK with enabling CPU_TURBO for all existing CPUFreqs